composer.json

```diff
@@ -16,32 +16,32 @@
         }
     ],
     "require": {
-        "php": "^7.3|^8.0",
-        "doctrine/annotations": "^1",
-        "doctrine/cache": "^1",
-        "doctrine/dbal": "^2.13",
-        "doctrine/orm": "^2.6",
+        "php": "^8.0",
+        "doctrine/annotations": "^1.13",
+        "doctrine/cache": "^2.1",
+        "doctrine/dbal": "^3.3",
+        "doctrine/orm": "^2.11",
         "doctrine/persistence": "^1.3.5|^2.0",
-        "illuminate/auth": "^8.0",
-        "illuminate/console": "^8.0",
-        "illuminate/container": "^8.0",
-        "illuminate/contracts": "^8.0",
-        "illuminate/pagination": "^8.0",
-        "illuminate/routing": "^8.0",
-        "illuminate/support": "^8.0",
-        "illuminate/validation": "^8.0",
-        "illuminate/view": "^8.0",
+        "illuminate/auth": "^9.0",
+        "illuminate/console": "^9.0",
+        "illuminate/container": "^9.0",
+        "illuminate/contracts": "^9.0",
+        "illuminate/pagination": "^9.0",
+        "illuminate/routing": "^9.0",
+        "illuminate/support": "^9.0",
+        "illuminate/validation": "^9.0",
+        "illuminate/view": "^9.0",
         "symfony/serializer": "^2.7|^3.0|^4.0|^5.0",
```

Comment:
This should probably be bumped to `^6.0` if you plan on only supporting Laravel 9.